packages/nextjs/test/integration/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "alwaysStrict": true,
```

Comment:
Why not extend `../../../../tsconfig.json`?

Reply:
I didnt want to change anything about boilerplate app other than adding pages, so we can make drop-in replacements easily.